### PR TITLE
Service Page Border Variables

### PR DESCRIPTION
**Pull request by 3mkay**
Additional variable for one of the service page borders.

### CODE REVIEW

File: css/base/variables.css [hunk 1]
```diff
@@ -390,4 +390,7 @@ body {
 
   /* Leaflet Mapping */
   --leaflet-map-outline-stroke-colour: #38f;
+  
+  /* Service Landing Page */
+  --service-cta-border: var(--border-width);
```
Comment (markconroy) [inline css/base/variables.css]: I tihnk this should evaluate to `--border`, shouldn't it?

File: css/components/service-cta-block.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .service-cta-block__list {
   padding-left: 0;
   list-style: none;
-  border-bottom: var(--border);
+  border-bottom: var(--service-cta-border);
```
Comment (markconroy) [inline css/components/service-cta-block.css:4]: This is fine, but 'only' if `--service-cta-border` evaluates to `--border` (and then it also won't break existing sites.

File: css/components/teaser.css [hunk 3]
```diff
@@ -55,7 +55,7 @@
 .news-article .field--name-localgov-news-related > .field__item {
   margin-bottom: var(--vertical-rhythm-spacing);
   padding-bottom: var(--vertical-rhythm-spacing);
-  border-bottom: var(--border);
+  border-bottom: var(--service-cta-border);
```
Comment (markconroy) [inline css/components/teaser.css:58]: If we are going to set the border on teasers to something specific, let's called it `--border-teaser` or `--teaser-border`, and let that evaluate to `--border` in the `variables.css` file

**Review by markconroy — CHANGES_REQUESTED** [review]

File: css/base/variables.css (hunk 1 above)
Comment (3mkay) [reply css/base/variables.css]: Indeed. I missed the declaration of that in amongst the others. 

Any issue if I separate those meta variables out into their own section to make them more obvious?

Comment (markconroy) [reply css/base/variables.css]: Yes, work away, happy to look at a PR for that to see how it works.

Comment (3mkay) [reply css/base/variables.css]: It's only a small tweak, have included it in this PR [9ef8f6c](https://github.com/localgovdrupal/localgov_base/pull/200/commits/9ef8f6cd9b68aee15e11963eff159d97d82ed10d)

**Comment by markconroy** [comment]
@3mkay 

Looks like we have a few conflicts here since this PR was created. Can you look at those (I reckon they are quite small) and we'll get this merged asap then. Thanks.

**Pull request closed without merging**